crates/bevy_transform/src/lib.rs

```diff
@@ -5,169 +5,43 @@
     html_favicon_url = "https://bevyengine.org/assets/icon.png"
 )]
 
-pub mod commands;
 /// The basic components of the transform crate
 pub mod components;
+
+#[cfg(feature = "bevy-support")]
+pub mod commands;
+
+/// Transform related bundles
+#[cfg(feature = "bevy-support")]
+pub mod bundles;
+
+/// Transform related bundles
+#[cfg(feature = "bevy-support")]
+pub mod traits;
+
+/// Transform related plugins
+#[cfg(feature = "bevy-support")]
+pub mod plugins;
+
+#[cfg(feature = "bevy-support")]
```

Comment:
A simple description here would be nice.

Reply:
Added a short description